src/main/resources/com/google/api/codegen/nodejs/package.json.snip

```diff
@@ -1,10 +1,14 @@
 @snippet generate(metadata)
   {
-    "repository": "GoogleCloudPlatform/google-cloud-node",
-    "name": "{@metadata.identifier}",
+    "name": "@@google-cloud/{@metadata.identifier}",
```

Comment:
This should be based on if teh domain layer location set in teh gapic client is set to google-cloud.